src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common.Tooling/Workspaces/TestLanguageServerFeatureOptions.cs

```diff
@@ -45,4 +45,6 @@ internal class TestLanguageServerFeatureOptions : LanguageServerFeatureOptions
     public override bool IncludeProjectKeyInGeneratedFilePath => _includeProjectKeyInGeneratedFilePath;
 
     public override bool MonitorWorkspaceFolderForConfigurationFiles => _monitorWorkspaceFolderForConfigurationFiles;
+
+    public override bool UseRazorCohostServer => throw new System.NotImplementedException();
```

Comment:
Should we just go ahead and add an optional constructor param for this?

Reply:
So far I've only done it as needed. This was hardoded to return `false` in another PR to the feature branch